operator/src/test/java/io/strimzi/kafka/access/KafkaAccessReconcilerTest.java

```diff
@@ -71,7 +71,8 @@ public class KafkaAccessReconcilerTest {
 
     @BeforeEach
     void beforeEach() {
-        operator = new Operator(overrider -> overrider.withKubernetesClient(client));
+        operator = new Operator(overrider -> overrider.withKubernetesClient(client)
+                .withUseSSAToPatchPrimaryResource(false)); // Motivation: mock Kubernetes client doesn't fully support some SSA features.
```

Comment:
Do the tests fail with SSA enabled? What features doesn't it support?

If the operator itself will have SSA enabled we need to make sure we are testing sufficiently with SSA.

Reply:
@im-konge Would you have any ideas on why all of the tests here fail when SSA is enabled? As Kate says, if we have SSA enabled, we should be able to test without disabling. I'm not sure why they fail.

Reply:
TBH I didn't check why they fail with SSA enabled. I can check it but later.